onnxruntime/test/onnx/main.cc

```diff
@@ -505,6 +501,8 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
   broken_tests.insert({"tf_mobilenet_v1_1.0_224", "result mismatch"});
   broken_tests.insert({"mobilenetv2-1.0", "result mismatch"});
   broken_tests.insert({"candy", "result mismatch"});
+  broken_tests.insert({"range_float_type_positive_delta_expanded", "get unknown exception from MKLDNN EP"});
```

Comment:
For some reason , the "expanded" version of the `Range` tests (the ones with the function body expanded as a whole graph) fail when `--use_mkldnn` are turned on. The exception is not obvious. These tests are valid ONNX models and I validated that they run successfully on the CPU EP. As such these tests are automatically created whenever we construct an op with a function body as a sanity check for a backend to test the function body. These are different from the "non-expanded" range tests which have a single "Range" node in the test graph. 

Putting these 2 into exclusion as they are no high priority enough to warrant an investigation of the MKLDNN EP.